experiments/AMIP/moist_mpi_earth/coupler_utils/masker.jl

```diff
@@ -70,12 +70,21 @@ function combine_surfaces!(combined_field::Fields.Field, masks::NamedTuple, fiel
     end
 
 end
-nans_to_zero(v) = isnan(v) ? FT(0) : v
+
+"""
+    nans_to_zero(v)
+Converts all NaNs to zero of the same type and throws a warning.
+"""
+function nans_to_zero(v)
+    v = isnan(v) ? typeof(v)(0) : v
+    @warn "NaNs were detected and converted to zeros."
```

Comment:
Based on the way that this function is used, I would recommend not putting a warning here.

Reply:
Ok, thanks. So you would put it only in the `Regridder.jl` module

Reply:
@charleskawczynski , I have used a global flag. This way, the warning is printed only once at the end of the for loop where this function is called. This is the best solution that I could think of. (Removing the function altogether would have been more difficult because of its vectorized usage). 

It passes CI and is ready for review. Thanks!